src/pandas_profiling/profile_report.py

```diff
@@ -55,7 +56,7 @@ def __init__(
         sensitive: bool = False,
         dark_mode: bool = False,
         orange_mode: bool = False,
-        tsmode: bool = False,
```

Comment:
`tsmode` should not be an optional input. Please keep it as _False_. This was a requirement when the feature was implemented.